UnitTests/Helpers/Test_StreamHelper.cs

```diff
@@ -55,7 +56,8 @@ public class Test_StreamHelper
         [TestMethod]
         public async Task Test_StreamHelper_PackagedFile()
         {
-            Assert.IsTrue(await StreamHelper.IsPackagedFileExistsAsync(PackagedFilePath));
+            var packageFolder = Package.Current.InstalledLocation;
+            Assert.IsTrue(await StorageFileHelper.FileExistsAsync(packageFolder, PackagedFilePath));
```

Comment:
Do you mind using extensions instead? The unit tests are also used to demonstrate the feature